scripts/generators/patternfly-4-component/templates/component/component.tsx.hbs

```diff
@@ -0,0 +1,24 @@
+import React, { DetailedHTMLProps } from 'react';
+import styles from '@patternfly/patternfly-next/{{typeDir}}/{{componentName}}/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+const defaultProps = {
+  children: null,
+  className: ''
+};
+
+interface {{ componentName }}Props {
+  children?: any;
+  className ?: string;
+}
+
+const {{componentName}}: React.SFC <{{componentName}}Props> = (props) => (
```

Comment:
I actually just saw today on another project that `React.SFC` will be replaced by `React.FunctionComponent` in the future. Might be good to switch to that now. Also you might want to remove the space between `SFC` and `<` (`React.FunctionComponent<{{componentName}}Props>`) prettier will likely remove it after the fact, but it would be better to avoid needing the cleanup step.

Reply:
Looks good to me with the space removal - we should add an issue and make sure the FunctionComponent change above is good and make that change separately (to get this in and be consistent).

Reply:
@dgutride updated the PR to remove the spaces.